install-cockroachdb.md

```diff
@@ -22,31 +22,18 @@ $(document).ready(function(){
 });
 </script>
 
-<!--<script>
-$(document).ready(function(){
-$(".osbutton").click(function(){ $(".install").hide();});
-    $("#mac").click(function(){
-        $("#macinstall").show();
-    });
-    $("#linux").click(function(){
-        $("#linuxinstall").show();
-    });
-    $("#windows").click(function(){
-        $("#windowsinstall").show();
-    });
-});
-</script>-->
-
-<button id="mac">Mac</button>
+<button id="mac">Mac OS X</button>
 <button id="linux">Linux</button>
 <button id="windows">Windows</button>
 
+<!-- Install instructions for Mac OS X -->
 <div id=macinstall>
-<p>There are currently two ways to install CockroachDB locally on OSX:</p>
+<p>There are three ways to install CockroachDB locally on Mac OS X:</p>
```

Comment:
Thanks, @petermattis. The latter command worked for me. I'll put that option in the docs. When we get it into the main repo, I'll update the command. `brew install cockroachdb` would be so much prettier ;). Do we need a new issue for that work?